openpower/package/openpower-pnor/openpower-pnor.mk

```diff
@@ -106,6 +109,13 @@ define OPENPOWER_PNOR_INSTALL_IMAGES_CMDS
             dd if=$(STAGING_DIR)/pnor/$(BR2_OPENPOWER_PNOR_FILENAME) of=$(STAGING_DIR)/pnor/$(BR2_OPENPOWER_PNOR_UPDATE_FILENAME) bs=32M count=1; \
             $(INSTALL) $(STAGING_DIR)/pnor/$(BR2_OPENPOWER_PNOR_UPDATE_FILENAME) $(BINARIES_DIR); \
         fi
+
+        # If this is a VPNOR system, run the generate-squashfs command and
+        # create a tarball
+        if [ "$(BR2_BUILD_VPNOR)" == "y" ]; then \
+            PATH=$(PATH):$(HOST_DIR)/usr/bin $(HOST_DIR)/usr/bin/generate-squashfs -f $(STAGING_DIR)/pnor/$(BR2_OPENPOWER_PNOR_FILENAME).squashfs.tar $(STAGING_DIR)/pnor/$(BR2_OPENPOWER_PNOR_FILENAME); \
```

Comment:
I'm new to op-build so forgive me if I'm ignorant. I'm don't see what the problem is with this workflow. The .pnor created by the steps above is an input to the script for generating the squashfs/VPNOR file, so we have to run the PNOR steps first. The serialization is not needless. If it's necessary that I do this with a CSV description file, could you give me some more assistance on how to do that and integrate it with what we have now? Maybe a wiki? Maybe an example? How would I run a bash script using that program? It looks like it's for making FFS partitions.